### DISCUSSION TITLE
How do I refresh the view?

### DISCUSSION THREAD

**Discussion by wastee** [sxyazi/yazi#570] [1. Q&A] [answered]
If I delete a lot of files, those files that have already been deleted still appear in YAZI. I can go out and return to the folder to "refresh." However, it would be nice if there were another way.

**Comment by sxyazi** [comment]
Hi, what's your yazi version? Please do a `yazi -V` and paste the output here.

**Reply by wastee** [reply]
Hi. Version is yazi 0.1.5

**Reply by sxyazi** [reply, marked as answer]
Please try the latest v0.2.2 to see if it works for you, https://github.com/sxyazi/yazi/releases/tag/v0.2.2

**Reply by wastee** [reply]
Hi. I'm not sure if it's a bug, but when I delete files in the search view, they still appear after deletion. In the "outside" normal view, everything is okay.

The version is latest v0.2.2

**Reply by sxyazi** [reply]
Yup, deleting files in the temporary view of search results has not been implemented yet, see https://github.com/sxyazi/yazi/issues/62, and I will attempt to implement it in the next target.
